src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1693,7 +1698,7 @@ internal void EnsureAnonymousTypeTemplates(CancellationToken cancellationToken)
 
                         if (success)
```

Comment:
Completing basically an incremental way to figure when we are done with all trees and signal to analyzers that compiler is all done.
At the level of Compilation there is just compiler and no notion of analyzers, so there is nothing to signal.